pkg/reaper/nodereaper/nodereaper.go

```diff
@@ -536,8 +539,11 @@ func (ctx *ReaperContext) reapOldNodes(gctx context.Context, w ReaperAwsAuth) er
 
 		// Skip if target node is self
 		if instance.NodeName == ctx.SelfNode {
-			log.Infof("self node termination attempted, skipping")
+			log.Infof("self node termination attempted, cordoning")
+
+			ctx.cordonNode(instance.NodeName, ctx.DryRun, ctx.IgnoreFailure)
 			ctx.labelNode(instance.NodeName, "governor.keikoproj.io/status", "current-node")
```

Comment:
Do we use this in scheduling? If not, we should probably start doing so, so that jobs do not get rescheduled onto the nodes.

Reply:
We don't (not sure the purpose of this, iirc from upstream), but scheduling will be blocked by cordoning anyway - node reaper does not tolerate `NoSchedule`

Reply:
Turns out we use it in our chart, so it's not supposed to schedule on these nodes 🤔